core/src/main/java/org/apache/accumulo/core/file/rfile/RelativeKey.java

```diff
@@ -467,24 +467,24 @@ private static byte[] readPrefix(DataInput in, ByteSequence prefixSource) throws
     return data;
   }
 
-  private static void readPrefix(DataInput in, MutableByteSequence dest, ByteSequence prefixSource)
+  private static void readPrefix(DataInput in, ArrayByteSequence dest, ByteSequence prefixSource)
       throws IOException {
     int prefixLen = WritableUtils.readVInt(in);
     int remainingLen = WritableUtils.readVInt(in);
     int len = prefixLen + remainingLen;
+    byte[] buf = dest.getBackingArray();
     if (dest.getBackingArray().length < len) {
```

Comment:
Could change this to `buf.length` like the code above.

Reply:
Done.